src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -8,7 +8,8 @@
    [broadfcui.common.icons :as icons]
    [broadfcui.common.modal :as modal]
    [broadfcui.common.style :as style]
-   [broadfcui.common.table :refer [Table]]
+   [broadfcui.common.table.table :refer [Table]]
+   [broadfcui.common.table.style :as table-style]
```

Comment:
sorry this going to be super duper nitpicky, but these should all be in alphabetical order. Would you mind rearranging?